### PR TITLE
chore(release): 6.2.1-alpha.3

### PR DESCRIPTION
**Pull request by SyntaxColoring**
# Overview

Update version specifiers, autogenerated changelogs, and human-written release notes in preparation for the v6.2.1-alpha.3 release.

# Review requests

Do the human-written updates to `api/release-notes.md` and `app-shell/build/release-notes.md` look good?

### CODE REVIEW

File: api/release-notes.md [hunk 1]
```diff
@@ -13,8 +13,9 @@ Welcome to the v6.2.1 release of the OT-2 software! This hotfix release addresse
 ### Bug Fixes
 
 - When you upload a protocol file larger than 2 megabytes, you will no longer get an error saying "Protocol run could not be created on the robot."
-- When you upload a protocol to an OT-2 that already has a lot of big protocols stored, the OT-2 will no longer show connection errors and become unresponsive for several minutes.
-- When a Thermocycler GEN2 is run for 50 days without a power cycle, it will no longer miscalculate hold times.
+- When you upload a protocol to an OT-2 that already has a lot of big protocols stored, the OT-2 is now less likely to show connection errors.
+- When you upload a Protocol Designer protocol that has many steps, the OT-2 is now less likely to show connection errors.
```
Comment (ecormany) [inline api/release-notes.md:17]: Good to spell out the delta in changes between alphas here, but for release we could consolidate these three down to something like `Improved performance of uploading and storing large protocols on the OT-2.`

File: app-shell/build/release-notes.md [hunk 2]
```diff
@@ -13,6 +13,7 @@ Welcome to the v6.2.1 release of the Opentrons App!
 ## Bug Fixes
 
 - Fixed a problem where certain kinds of robot errors would block you from accessing the robot's settings.
+- Improved the time it takes to load a device's details page.
```
Comment (ecormany) [inline app-shell/build/release-notes.md:16]: I would move this to a `## Improved Features` section and edit to `The device details page now loads more quickly.`

**Review by ecormany — APPROVED** [review]
Good for alpha. We'll give release notes a final touch-up before stable.

File: app-shell/build/release-notes.md (hunk 2 above)
Comment (ecormany) [reply app-shell/build/release-notes.md:16]: …or `faster`, `fast` is a perfectly cromulent adverb.

File: api/release-notes.md (hunk 1 above)
Comment (SyntaxColoring) [reply api/release-notes.md:17]: Done in #12047.

File: app-shell/build/release-notes.md (hunk 2 above)
Comment (SyntaxColoring) [reply app-shell/build/release-notes.md:16]: Done in #12047.

**Review by nusrat813 — APPROVED** [review]
LGTM once test pass

**Pull request merged**